api/src/certification/configuration/application/sco-blocked-access-dates-route.js

```diff
@@ -0,0 +1,67 @@
+import JoiDate from '@joi/date';
+import BaseJoi from 'joi';
+
+const Joi = BaseJoi.extend(JoiDate);
+
+import { securityPreHandlers } from '../../../shared/application/security-pre-handlers.js';
+import { scoBlockedAccessDatesController } from './sco-blocked-access-dates-controller.js';
+
+const register = async function (server) {
+  server.route([
+    {
+      method: 'PATCH',
+      path: '/api/admin/sco-blocked-access-dates/{key}',
+      config: {
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.hasAtLeastOneAccessOf([securityPreHandlers.checkAdminMemberHasRoleSuperAdmin])(
+                request,
+                h,
+              ),
+            assign: 'hasAuthorizationToAccessAdminScope',
+          },
+        ],
+        validate: {
+          params: Joi.object({
+            key: Joi.string().required(),
```

Comment:
Je propose de typer un peu + la vérification, mais ca nécessite de créer un Enum côté Domain qui sera basiquement un sous-ensemble de la version + large de `api/src/organizational-entities/domain/models/Tag.js`

```suggestion
            key: Joi.string().required()..valid(...Object.values(Tags)),
```